test/rules/base.jl

```diff
@@ -1,10 +1,31 @@
-function test_scalar(f, f′, xs...)
+_isapprox(x, y; kwargs...) = isapprox(x, y; kwargs...)
+
+const AbstractDifferential = ChainRules.AbstractDifferential
+
+_isapprox(x::AbstractDifferential, y; kwargs...) = _isapprox(extern(x), y; kwargs...)
+_isapprox(x, y::AbstractDifferential; kwargs...) = _isapprox(x, extern(y); kwargs...)
+_isapprox(x::AbstractDifferential, y::AbstractDifferential; kwargs...) = _isapprox(extern(x), extern(y); kwargs...)
+
+_isapprox(x::Wirtinger, y; kwargs...) = _isapprox(x.primal, y; kwargs...) &&
+    _isapprox(x.conjugate, Zero(); kwargs...)
+_isapprox(x, y::Wirtinger; kwargs...) = _isapprox(x, y.primal; kwargs...) &&
+    _isapprox(Zero(), y.conjugate; kwargs...)
+_isapprox(x::Wirtinger, y::Wirtinger; kwargs...) = _isapprox(x.primal, y.primal; kwargs...) &&
+    _isapprox(x.conjugate, y.conjugate; kwargs...)
+
+const ≈₁ = _isapprox
+
+function test_scalar(f, f′, xs...; broken = false)
```

Comment:
The `broken` keyword argument doesn't appear to actually be used; it should just be removed.

Reply:
Yes, that's just a remainder. I'll just delete it.

Reply:
Sounds good, thanks.